example/timer/odp_timer_accuracy.c

```diff
@@ -94,7 +110,7 @@ static void print_usage(void)
 	       "  -r, --res_ns <nsec>     Timeout resolution in nsec. Default: period / 10\n"
 	       "  -R, --res_hz <hertz>    Timeout resolution in hertz. Note: resolution can be set\n"
 	       "                          either in nsec or hertz (not both). Default: 0\n"
-	       "  -f, --first <nsec>      First timer offset in nsec. Default: 300 msec\n"
+	       "  -f, --first <nsec>      First timer offset in nsec. Default: 0 for mode 3, otherwise 300 msec\n"
 	       "  -x, --max_tmo <nsec>    Maximum timeout in nsec. When 0, max tmo is calculated from other options. Default: 0\n"
 	       "  -n, --num <number>      Number of timeout periods. Default: 50\n"
 	       "  -b, --burst <number>    Number of timers per a timeout period. Default: 1\n"
```

Comment:
Burst / burst_gap could document that in period mode first + burst*burst_gap must be less than period length

Reply:
v4.